App.jsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
```

Comment:
Since you only have few routes, it is fine to leave it in app.jsx file. With more routing, best practice would be moving routes to a router file.